tests/rustdoc-json/attrs/optimize.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(optimize_attribute)]
+
+//@ is "$.index[?(@.name=='speed')].attrs" '["#[attr = Optimize(Speed)]"]'
```

Comment:
Is the display of the inner value supposed to be like that? Shouldn't it be `speed`?

Reply:
It's the debug printing of attributes, which mirrors their semantic strucure in HIR, not their semantic structure.

See https://doc.rust-lang.org/nightly/nightly-rustc/rustc_attr_data_structures/enum.OptimizeAttr.html and https://doc.rust-lang.org/nightly/nightly-rustc/rustc_attr_data_structures/enum.AttributeKind.html#variant.Optimize

Reply:
I see. How attributes are displayed is incoherent though, which isn't great. For example:

```
attrs/repr_align.rs:3://@ is "$.index[?(@.name=='Aligned')].attrs" '["#[repr(align(4))]"]'
attrs/export_name_2024.rs:7://@ is "$.index[?(@.name=='example')].attrs" '["#[export_name = \"altered\"]"]'
attrs/repr_c.rs:13://@ is "$.index[?(@.name=='ReprCUnion')].attrs" '["#[repr(C)]"]'
attrs/repr_packed.rs:6://@ is "$.index[?(@.name=='Packed')].attrs" '["#[repr(packed(1))]"]'
keyword_private.rs:8://@ is   "$.index[?(@.name=='foo')].attrs" '["#[doc(keyword = \"match\")]"]'
```

Some of them are "code-like" and other represent the internal type representation in rustc, which is subject to change. Do we plan to uniform it at some point?